sql: nightly benchmark broken

The nightly benchmark seems to have stalled during the `sql.test` portion. The `sql.test.stderr` log is growing forever with the following log lines:

```
I160727 10:14:34.148590 server/status/runtime.go:189  runtime stats: 1.5 GiB RSS, 92 goroutines, 20 MiB/1.1 GiB/1.2 GiB GO alloc/idle/total, 81 MiB/147 MiB CGO alloc/total, 205.60c
go/sec, 0.03/0.01 %(u/s)time, 0.00 %gc (1x)
W160727 10:14:34.158639 ts/db.go:104  node unavailable; try another peer
W160727 10:14:34.158665 server/node.go:642  error recording status summaries: node unavailable; try another peer
W160727 10:14:42.055111 gossip/gossip.go:942  not connected to cluster; use --join to specify a connected node
I160727 10:14:44.028505 server/status/runtime.go:189  runtime stats: 1.5 GiB RSS, 94 goroutines, 21 MiB/1.1 GiB/1.2 GiB GO alloc/idle/total, 81 MiB/147 MiB CGO alloc/total, 215.81c
go/sec, 0.02/0.01 %(u/s)time, 0.00 %gc (0x)
```

The test invocation on TeamCity is here: https://teamcity.cockroachdb.com/viewLog.html?buildId=5068&buildTypeId=Cockroach_BenchmarkTests&tab=buildLog
he nightly benchmark seems to have stalled during the `sql.test` portion. The `sql.test.stderr` log is growing forever with the following log lines:

```
I160727 10:14:34.148590 server/status/runtime.go:189  runtime stats: 1.5 GiB RSS, 92 goroutines, 20 MiB/1.1 GiB/1.2 GiB GO alloc/idle/total, 81 MiB/147 MiB CGO alloc/total, 205.60c
go/sec, 0.03/0.01 %(u/s)time, 0.00 %gc (1x)
W160727 10:14:34.158639 ts/db.go:104  node unavailable; try another peer
W160727 10:14:34.158665 server/node.go:642  error recording status summaries: node unavailable; try another peer
W160727 10:14:42.055111 gossip/gossip.go:942  not connected to cluster; use --join to specify a connected node
I160727 10:14:44.028505 server/status/runtime.go:189  runtime stats: 1.5 GiB RSS, 94 goroutines, 21 MiB/1.1 GiB/1.2 GiB GO alloc/idle/total, 81 MiB/147 MiB CGO alloc/total, 215.81c
go/sec, 0.02/0.01 %(u/s)time, 0.00 %gc (0x)
```

The test invocation on TeamCity is here: https://teamcity.cockroachdb.com/viewLog.html?buildId=5068&buildTypeId=Cockroach_BenchmarkTests&tab=buildLog


UPDATE: current issue is that `sql.BenchmarkPgbenchExec_{Cockroach,Postgres}` fail in the absence of `pgbench`, which we do not install anywhere.

--- a/storage/queue_test.go
+++ b/storage/queue_test.go
@@ -356,7 +356,6 @@ func TestBaseQueueAddRemove(t *testing.T) {
 // rejected when the queue has 'acceptsUnsplitRanges = false'.
 func TestAcceptsUnsplitRanges(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("#8001")
 	s, _, stopper := createTestStore(t)
 	defer stopper.Stop()
 
@@ -399,10 +398,15 @@ func TestAcceptsUnsplitRanges(t *testing.T) {
 	bq.Start(s.ctx.Clock, stopper)
 
 	// Check our config.
-	sysCfg, ok := s.ctx.Gossip.GetSystemConfig()
-	if !ok {
-		t.Fatal("config not set")
-	}
+	var sysCfg config.SystemConfig
+	util.SucceedsSoon(t, func() error {
+		var ok bool
+		sysCfg, ok = s.ctx.Gossip.GetSystemConfig()
+		if !ok {
+			return errors.New("system config not yet present")
+		}
+		return nil
+	})
 	neverSplitsDesc := neverSplits.Desc()
 	if sysCfg.NeedsSplit(neverSplitsDesc.StartKey, neverSplitsDesc.EndKey) {
 		t.Fatal("System config says range needs to be split")

